x-pack/metricbeat/module/mssql/doc.go

```diff
@@ -2,5 +2,5 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
-// Package foo is a Metricbeat module that contains MetricSets.
-package foo
+// Package mssql is a Metricbeat module that contains MetricSets.
```

Comment:
Was this moved on purpose? I would assume both files should exist?

Reply:
Sorry, I think this have been my git tool that detected a rename instead of an single addition because I really didn't do this manually. I will fix it in the next commit